src/controller/ApplicationController.tsx

```diff
@@ -280,7 +280,7 @@ class ApplicationController<
  */
 function StartControllerScope<
   T extends ApplicationControllerConstructor<any>,
-  C extends ComponentType<any>,
+  C extends ComponentType<any> & ApplicationView,
```

Comment:
Would this be a more accurate type:

```suggestion
  C extends ComponentType<any> | ComponentType<any> & ApplicationView,
```